pkg/grpc/proxy/handler.go

```diff
@@ -148,29 +163,30 @@ func (s *handler) handler(srv interface{}, serverStream grpc.ServerStream) error
 				if err != nil {
 					return struct{}{}, err
 				}
+			} else {
+				return struct{}{}, err
```

Comment:
Reviewer: this fixes a panic if `grpc.NewClientStream` returns an error that is not one of those handled above